examples/java/src/test/java/dev/selenium/getting_started/UsingSeleniumTest.java

```diff
@@ -1,38 +1,65 @@
 package dev.selenium.getting_started;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 
-import java.time.Duration;
+public class UsingSeleniumTest {
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+	WebDriver driver;
 
-public class UsingSeleniumTest {
+	@BeforeAll
+	public static void beforeAll() {
+		System.out.println("This method will be executed only once before all the tests.");
+	}
+
+	@BeforeEach
+	public void setup() {
+		System.out.println(
+				"  This method will be executed before each test, so here write the common code that has to be executed.");
+		driver = new ChromeDriver();
+		driver.manage().window().maximize();
+		driver.manage().timeouts().implicitlyWait(Duration.ofMillis(500));
+		driver.get("https://www.selenium.dev/selenium/web/web-form.html");
```

Comment:
This should be part of the test, not the setup.

Reply:
This code has been moved from setup to test body.